test/ringpop-assert.js

```diff
@@ -224,13 +224,13 @@ function changeStatus(t, tc, sourceIx, subjectIx, status, subjectIncNoDelta) {
     return f;
 }
 
-function sendPing(t, tc, nodeIx, piggybackOpts) {
+function sendPing(t, tc, nodeIx, piggybackOpts, bodyOverrides) {
```

Comment:
As It is getting hard to understand what the function is doing from the docstring + signature, please document `piggybackOpts` and `bodyOverrides`.